packages/thirdweb/src/react/core/hooks/pay/useBuyWithCryptoQuote.ts

```diff
@@ -80,6 +80,7 @@ export function useBuyWithCryptoQuote(
   return useQuery({
     ...queryParams,
     queryKey: ["buyWithCryptoQuote", params],
+    refetchInterval: 30_000,
```

Comment:
This probably fixes everything tbh. Just make it 20sec to make sure it's fresh

Would remove the other refetches